docs/SearchBar.md

```diff
@@ -37,6 +37,7 @@ Then, add `search-bar` block into your app theme, as we do in our [Store Header]
 | `openAutocompleteOnFocus` | `Boolean`                                     | Identify if autocomplete should be open on input focus or not.                                                                                                                                    | `false`        |
 | `placeholder`             | `String!`                                     | Placeholder to be used on the input                                                                                                                                                               | -              |
 | `inputType` | `enum` | Defines the value for the `type` HTML attribute (from the `<input>` field). Possible values are: `search` and `text`. We strongly recommend you to use `search` as it fixes some iOS issues. The `text` value is only kept as default for backwards compatibility. | `text` |
+| `containerMode` | `String` | Allows the autocomplete component to be displayed with or without overlap. The available properties are `overlay` and container | `overlay`
```

Comment:
```suggestion
| `containerMode` | `enum` | Defines how the autocomplete component should be displayed. Possible values are: `overlay` (suggestions overlapping other components) and `container` (displays the suggestion within a container). | `overlay` |
```

Reply:
Hey @weslybrandao! I see here that my previous suggestion was labeled as resolved, but it was not applied to the docs. Don't know if it was purposely, but I'm submitting it again for your review :)

Reply:
Hello @guerreirobeatriz, sorry, I ended up not paying attention to your suggestion and made another commit, thank you very much for sending it again for review, I believe it is now ok =)